translations/bnpchs-final-2019-makhluk-kaki-banyak.html

```diff
@@ -0,0 +1,107 @@
+<style>
+  table, th, td{
+  border: 1px solid black;
+  border-collapse: collapse;
+}
+</style>
```

Comment:
Remove these styles

Reply:
itu untuk tabelnya, I have no idea how to add borders on the table without styles